editor/editor/editor.php

```diff
@@ -76,14 +76,15 @@ private function getMode( $postType ) {
 		}
 	}
 
-	public function getClientConfig( $context ) {
+	public function getClientConfig( $config, $context ) {
 		$config = [
 			'hash'          => wp_create_nonce( Brizy_Editor_API::nonce ),
 			'editorVersion' => BRIZY_EDITOR_VERSION,
 			'url'           => set_url_scheme( admin_url( 'admin-ajax.php' ) ),
 			'actions'       => $this->getApiActions(),
-			'pageId'        => $this->post->getWpPostId()
-		];
+			'pageId'        => $this->post->getWpPostId(),
+            'collectionTypes' => $config['wp']['postLoopSources']
```

Comment:
Baga la incputu functiei ceva de genu:

```php
if(!isset($config['wp']['postLoopSources'])) {
        throw new \Exception("Unable to locate the $config['wp']['postLoopSources']");
}
```


Simt ca cineva o sa mute postLoopSources in alt parte si aici o sa moara tot.